infrastructure/prx_models/models/aruco_mip_36h12/empty_aruco.xml

```diff
@@ -0,0 +1,22 @@
+<mujoco model="empty_aruco">
+  <compiler angle="radian" />
+  <size njmax="500" nconmax="100"/>
+  <option timestep="0.001" integrator="Euler"/>
```

Comment:
When `mujoco` includes a file, I am unsure about the behavior of the `option`s, i.e. whether the base XML's options override the includes or not (it shouldn't). Or we can improve the `option`, etc., from this file (from what I can tell, this is only defining the tag bodies, so physics is not very important).

Reply:
I was using this to test the aruco tags _in isolation_. This shouldn't be imported into another mj xml...

Reply:
I checked on 3.0.0 and it seems to happen -- can you check on your version, and remove if unnecessary?